### ISSUE TITLE
No names for people without names

### ISSUE THREAD

**Issue by turt2live** [t3chguy/matrix-static#34]
Should probably fall back to something, otherwise the list looks weird: https://view.matrix.org/room/!zxWLnBMJtBomyQlnRV:tang.ents.ca/members?page=1 (see bottom or page 2)

**Comment by t3chguy** [comment]
Yeah the table layout is temporary, I can see it in the future being Display Name if one with a subtext of the MXID, or just MXID if no Display Name
